crates/web5/src/crypto/dsa/secp256k1.rs

```diff
@@ -3,9 +3,12 @@ use crate::errors::Result;
 use crate::errors::Web5Error;
 use base64::{engine::general_purpose, Engine as _};
 
+#[cfg(test)]
 pub struct Secp256k1Generator;
 
+#[cfg(test)]
 impl Secp256k1Generator {
+    #[cfg(test)]
```

Comment:
If this is meant to be part of the public API, why only build it for testing?

Reply:
yup, done!

Reply:
fyi @diehuxx in [the commit here](https://github.com/TBD54566975/web5-rs/pull/300/commits/89a7bd71a8ef93678677ccb3c750c3878be92533) I had to make the secp256k1 mod fully public because CI is setup to fail on warnings https://github.com/TBD54566975/web5-rs/actions/runs/10497204019/job/29079498494

just a heads up to that latest change